ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -102,9 +109,16 @@ if [[ -s "${SHARED_DIR}/xpn.json" ]]; then
   HOST_PROJECT_COMPUTE_SERVICE_ACCOUNT="$(jq -r '.computeServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_CONTROL_SERVICE_ACCOUNT="$(jq -r '.controlServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_PRIVATE_ZONE_NAME="$(jq -r '.privateZoneName' "${SHARED_DIR}/xpn.json")"
+
+  if [[ -v GOOGLE_CLOUD_XPN_SA ]]; then
+    echo "Using XPN configurations..."
+    PROJECT_OPTION="--project ${HOST_PROJECT} --account ${GOOGLE_CLOUD_XPN_SA}"
+  else
+    PROJECT_OPTION="--project ${HOST_PROJECT}"
+  fi
 else
```

Comment:
Nit: Probably can avoid the two else cases to make it look neater by setting HOST_PROJECT and PROJECT_OPTION before the if blocks
```
HOST_PROJECT="${PROJECT_NAME}"
PROJECT_OPTION="--project ${HOST_PROJECT}"
...
  if [[ -v GOOGLE_CLOUD_XPN_SA ]]; then
    echo "Using XPN configurations..."
    PROJECT_OPTION="${PROJECT_OPTION} --account ${GOOGLE_CLOUD_XPN_SA}"
  fi
```